services/user.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
```

Comment:
User = get_user_model()

Reply:
I've done it but I am not sure if I understood you correctly